libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -1405,7 +1409,7 @@ void AP_GPS::send_mavlink_gps_raw(mavlink_channel_t chan)
 void AP_GPS::send_mavlink_gps2_raw(mavlink_channel_t chan)
 {
     // always send the message if 2nd GPS is configured
-    if (params[1].type == GPS_TYPE_NONE) {
+    if (params[1].type == GPS_TYPE_NONE || drivers[1] == nullptr) {
```

Comment:
```suggestion
    if (params[1].type == GPS_TYPE_NONE) {
```

per referenced PR, this is on purpose

Reply:
I will undo it.